### PR TITLE
feat: add notification component for legitimacy of the data purposes.

### PR DESCRIPTION
**Pull request by JHNLWHD**
Re: #2 

This PR includes:
- Creation of notification component.
- Links of the external sources.

### CODE REVIEW

File: components/Notification.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<template>
+  <section
+    class="flex items-center justify-between flex-no-wrap bg-blue-100 p-6"
```
Comment (jbienesdev) [inline components/Notification.vue]: you can remove `items-center justify-between flex-no-wrap`. no effect to overall styling.

File: components/Notification.vue [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  >
+    <div class="flex items-center flex-shrink-0 text-blue-500 mr-6">
```
Comment (jbienesdev) [inline components/Notification.vue]: remove `flex-shrink-0`. gets cut off because of this class.

covid

File: pages/index.vue [hunk 3]
```diff
@@ -1,35 +1,40 @@
...
-        :decrease="184"
-      />
-      <CardCounter
-        legend-color="red"
-        card-title="Case Confirmed"
-        :total-cases="1"
-        :increase="1"
-      />
-    </section>
+  <div class="max-w-full m-auto">
```
Comment (jbienesdev) [inline pages/index.vue]: You can remove all these classes. Block elements like `div` has a full width by default.

File: tailwind.config.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
-** TailwindCSS Configuration File
-**
-** Docs: https://tailwindcss.com/docs/configuration
-** Default: https://github.com/tailwindcss/tailwindcss/blob/master/stubs/defaultConfig.stub.js
-*/
+ ** TailwindCSS Configuration File
+ **
+ ** Docs: https://tailwindcss.com/docs/configuration
+ ** Default: https://github.com/tailwindcss/tailwindcss/blob/master/stubs/defaultConfig.stub.js
+ */
```
Comment (jbienesdev) [inline tailwind.config.js]: remove space. not part of this commit.

File: components/Notification.vue (hunk 1 above)
Comment (JHNLWHD) [reply components/Notification.vue]: thanks for the heads up. already removed it.

File: components/Notification.vue (hunk 2 above)
Comment (JHNLWHD) [reply components/Notification.vue]: forgot to remove this specific class `flex-shrink-0` after doing some experiments. thanks for testing.

File: pages/index.vue (hunk 3 above)
Comment (JHNLWHD) [reply pages/index.vue]: today I learned. thanks for this info.
